src/cpp/session/modules/rmarkdown/NotebookExec.cpp

```diff
@@ -320,6 +320,7 @@ void ChunkExecContext::connect()
    if (error)
       LOG_ERROR(error);
 
+   setExecuting(true);
```

Comment:
As I read the code, this won't get called for the `executeAlternateEngineChunk` method since it does not use a ChunkExecContext. Don't we also want to set the Executing flag for that code path?

Reply:
Ah, you are right -- and this also points out that I had flipped the conditional here: https://github.com/rstudio/rstudio/pull/14368/files#diff-8c8a0cf2f04c6ee34c5e8f845fa44c02249ec41d177b56af8644f7a54887632eR243